### PR TITLE
Cart Management

### PR DESCRIPTION
**Pull request by NtshVrm**
Hey please review the cart management feature
Thanks!

- Add to cart (Only after the user is logged in)
- Remove from cart
- Updating quantity of product in cart

### CODE REVIEW

**Review by md-amir-gauhar — APPROVED** [review]
Looks good to me

File: src/components/productListing/ProductCard.jsx [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
+
+  useEffect(() => {
+    localStorage.getItem("login") ? setTokenState(true) : setTokenState(false);
+  });
+
+  const { _id } = item;
+
+  const inCart = cart?.find((obj) => obj._id === _id);
+
+  // console.log(inCart);
```
Comment (Pratik1005) [inline src/components/productListing/ProductCard.jsx]: Remove commented console log

File: src/components/productListing/ProductCard.jsx [hunk 2]
```diff
@@ -45,11 +98,14 @@ export function ProductCard({ item, type }) {
       </div>
       <div className="add-to-cart">
         <button
-          className={`card-button ${cartStatus ? "remove" : ""}`}
-          onClick={() => setcartStatus((prev) => !prev)}
+          className={`card-button ${tokenState && inCart ? "remove" : ""}`}
+          onClick={() =>
+            // setcartStatus((prev) => !prev)
```
Comment (Pratik1005) [inline src/components/productListing/ProductCard.jsx]: Remove commented code

**Review by Pratik1005 — APPROVED** [review]
Rest looks good

**Pull request merged**